frontend/src/components/MissionOverview/OngoingMissionView.tsx

```diff
@@ -20,13 +21,22 @@ const OngoingMissionSection = styled.div`
 export function OngoingMissionView() {
     const apiCaller = useApi()
     const [ongoingMissions, setOngoingMissions] = useState<Mission[]>([])
+    const [pausedMissions, setPausedMissions] = useState<Mission[]>([])
+    const [missionsToDisplay, setMissionsToDisplay] = useState<Mission[]>([])
     useEffect(() => {
         apiCaller.getMissionsByStatus(MissionStatus.Ongoing).then((missions) => {
             setOngoingMissions(missions)
         })
+        apiCaller.getMissionsByStatus(MissionStatus.Paused).then((pausedMissions) => {
+            setPausedMissions(pausedMissions)
+        })
     }, [])
+    useEffect(() => {
+        const missions: Mission[] = ongoingMissions.concat(pausedMissions)
+        setMissionsToDisplay(missions)
+    }, [ongoingMissions, pausedMissions])
```

Comment:
All this is a symptom of the missing "filter functionality" in the backend. It should be possible to query the backend with a filter on several mission statuses, and thus it should not be necessary to do this double collection and concat and stuff

Reply:
But for this PR this is a good solution!